Wave1_data/1.preprocess_data/scripts/1.define_platemap.py

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# This notebook extracts a platemap of the data
+
+# In[1]:
+
+
+import pathlib
+
+import pandas as pd
+
+# In[2]:
+
+
+# absolute path to the raw data directory only works on this machine
+path_to_plate_map = pathlib.Path("../../../data/raw/platemaps/").resolve()
+path_to_plate_map.mkdir(parents=True, exist_ok=True)
+path_to_plate_map = path_to_plate_map / "wave1_plate_map.csv"
+
+
+# In[3]:
+
+
+dict_platemap = {
```

Comment:
This dictionary seems short compared to the other dictionary made for the JSON file in the last notebook. Are not all wells used?

Reply:
Recommend code comment here to provide any clarity or context.

Reply:
added comments